compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/SyntaxKind.java

```diff
@@ -517,4 +517,8 @@
     public String stringValue() {
         return strValue;
     }
+
+    public boolean isKeyword() {
```

Comment:
This is a public API. Is this intentional?

Reply:
Removed with d2b4306.